packages/react-scripts/scripts/init.js

```diff
@@ -134,6 +171,10 @@ module.exports = function(
     }
   }
 
+  if (gitInit()) {
+    console.log('Initializing git repository');
```

Comment:
this is done after git initialized so i guess it's better to say git repository initialized rather than initializing

Reply:
👍